auto_round_extension/torch/qlinear_int32_zp.py

```diff
@@ -0,0 +1,275 @@
+import math
+from logging import getLogger
+
+import numpy as np
```

Comment:
Add license. If a significant portion of the code is derived from GPTQModel, be sure to include their license as well.